src/pages/events.js

```diff
@@ -1,38 +1,47 @@
 import React, { Component } from 'react'
 import Link from 'gatsby-link'
+import PropTypes from 'prop-types'
 import EventListingCard from '../components/EventListingCard'
 
 export default class Events extends Component {
-  state = {
-    events: this.props.data.allContentfulEvent.edges
+  constructor(props) {
```

Comment:
why did you switch away from the property initializer syntax?

Reply:
According to React docs when initialising state with data from props, the constructor should be used. 

Also, the linter was also throwing an error because it didn't recognise the `=` as a syntactically valid symbol. how weird

Reply:
As far as I'm aware, property initializers are fine for setting the initial state value because in practice the initialization logic is hoisted into the constructor. See, e.g. https://stackoverflow.com/a/37788410/1382997

I think the syntax error was just because we don't have the babel-eslint plugin installed: https://github.com/babel/babel-eslint

Reply:
will revert the constructor changes and using babel-eslint as the parser works for that error